src/languages/css.js

```diff
@@ -21,7 +27,10 @@ export default function(hljs) {
         ]
       }
     ]
-  }
+  };
+  var VENDOR_PREFIX= {
+    begin: /-(webkit|moz|ms|o)-/
+  };
```

Comment:
A candidate for `css-shared`.

Reply:
Yeah, that's why I created a branch for this.  It's going to take some time to work thru it all.